internal/doc/butane.yaml

```diff
@@ -344,6 +344,10 @@ root:
               desc: sets the minimum number of pieces required to decrypt the device. Default is 1.
             - name: discard
               desc: whether to issue discard commands to the underlying block device when blocks are freed. Enabling this improves performance and device longevity on SSDs and space utilization on thinly provisioned SAN devices, but leaks information about which disk blocks contain data. If omitted, it defaults to false.
+            - name: cex 
+              desc: whether or not to use cex device.
+              children:
+                - name: enabled
```

Comment:
Missing the description here

Reply:
HI @travier , Oh, My Bad, I verified only from the docs side. Now i've added the description.